02-spatial-data.Rmd

```diff
@@ -253,36 +269,38 @@ plot(world_centroids, add = TRUE, cex = world$pop / 1e8, lwd = 3)
 
 To understand how the spatial components of simple features work, it is vital to understand simple feature geometries.
-For this reason we cover each type currently supported `sfg` in the next sections before moving to describe how they can be combined to form `sfc` and eventually full `sf` objects.
+For this reason we cover each currently supported `sfg` type in the next subsections before moving on to describe how these can be combined to form `sfc` and eventually full `sf` objects.
 
 #### Simple feature geometry types {#geometry}
 
 <!-- This section demonstrates how the full range of geometry types supported by the **sf** package can be created, combined and plotted. -->
 Geometries are the basic building blocks of simple features.
-Simple features could be represented as one of the 17 geometry types using the **sf** package.
-In this chapter we will focus on seven, the most commonly used, simple features types: `POINT`, `LINESTRING`, `POLYGON`, `MULTIPOINT`, `MULTILINESTRING`, `MULTIPOLYGON` and `GEOMETRYCOLLECTION`.<!--FIG-->
-The whole list of possible feature types could be found in [the PostGIS manual ](http://postgis.net/docs/using_postgis_dbmanagement.html).
-
-Simple features, outside of an R environment, could be represented in one of two ways, ether as a well-known binary (WKB) or well-known text (WKT). 
-Well-known binary (WKB) representations are usually hexadecimal strings, which are used to transfer and store geometry objects in databases.
-Well-known text (WKT), on the other hand, is a text markup description of simple features. 
-Both formats are exchangeable, therefore we would focus only on the well-known text (WKT) representation.
-
-All of the geometry types are built of points. 
-Each point could be described as coordinates in a 2D, 3D or 4D space:
-
-- X and Y coordinates represent location of points. It could be either easting and northing or longitude and latitude.
-- A Z coordinate denotes altitude.
+Simple features in R can take on one of the 17 geometry types supported by the **sf** package.
+In this chapter we will focus on the seven most commonly used types: `POINT`, `LINESTRING`, `POLYGON`, `MULTIPOINT`, `MULTILINESTRING`, `MULTIPOLYGON` and `GEOMETRYCOLLECTION`.<!--FIG-->
+Find the whole list of possible feature types in [the PostGIS manual ](http://postgis.net/docs/using_postgis_dbmanagement.html).
+
+Generally, well-known binary (WKB) or well-known text (WKT) are the standard encoding for simple feature geometries.
+WKB representations are usually hexadecimal strings easible readable for computers.
+This is why, GIS and spatial databases use WKB to transfer and store geometry objects.
+WKT, on the other hand, is a human-readable text markup description of simple features. 
+Both formats are exchangeable, and if we present one, we will naturally choose the WKT representation.
+
+The basis for each of the geometry types is the point. 
```

Comment:
I didn't write this but agree that it's not the most enlightening content of this chapter and suggest we can delete this for now. It also seems to duplicate stuff at the beginning of the chapter where we plot a point in Euclidean space. In summary, agree: delete (heads-up @Nowosad we can always re-add this).